scripts/solutions.sh

```diff
@@ -20,6 +20,8 @@ SOLUTION=''
 VERBOSE=${VERBOSE:-0}
 VERSION=''
 
+export KUBECONFIG="$KUBECONFIG"
```

Comment:
`export KUBECONFIG` should be enough (and more portable).

Reply:
Ok, I see

Done!